Wrappers/Python/ccpi/optimisation/functions/L2NormSquared.py

```diff
@@ -114,41 +152,31 @@ def proximal(self, x, tau, out = None):
                 x.divide((1+2*tau), out=out)
 
     
-    def proximal_conjugate(self, x, tau, out=None):
-        
-        r'''Proximal operator of the convex conjugate of L2NormSquared at x:
-           
-           .. math::  prox_{\tau * f^{*}}(x)'''
-        
-        if out is None:
-            if self.b is not None:
-                return (x - tau*self.b)/(1 + tau/2) 
-            else:
-                return x/(1 + tau/2)
-        else:
-            if self.b is not None:
-                x.subtract(tau*self.b, out=out)
-                out.divide(1+tau/2, out=out)
-            else:
-                x.divide(1 + tau/2, out=out)
-                                        
-    def __rmul__(self, scalar):
-        
-        '''Multiplication of L2NormSquared with a scalar        
-           
-           Returns: ScaledFunction'''
-        
-        return ScaledFunction(self, scalar)  
-
-
+#    def proximal_conjugate(self, x, tau, out=None):
```

Comment:
same as the L1Norm, `proximal conjugate` and `mul` are from [Function](https://github.com/vais-ral/CCPi-Framework/blob/merge_master_into_Functions/Wrappers/Python/ccpi/optimisation/functions/Function.py)